web-bundle/src/main/java/com/graphhopper/replica/StreetEdgeExporter.java

```diff
@@ -34,7 +34,7 @@
     private static final List<String> HIGHWAY_FILTER_TAGS = Lists.newArrayList("bridleway", "steps");
     private static final List<String> INACCESSIBLE_MOTORWAY_TAGS = Lists.newArrayList("motorway", "motorway_link");
     private static final String[] COLUMN_HEADERS = {"stableEdgeId", "startVertex", "endVertex", "startLat", "startLon",
-            "endLat", "endLon", "geometry", "streetName", "distance", "osmid", "speed", "flags", "lanes", "highway"};
+            "endLat", "endLon", "geometry", "streetName", "distance", "osmid", "speed", "lanes", "highway"};
```

Comment:
Good call to look into this! I searched around and did find a good number of places where the flags are being used :( Sensor mapping, auto volume model, AADT processor...not ideal.

We definitely want to stop using these in the model repo ASAP (it's kind of scary how much they're being relied on), but I agree it might be a bigger task than we realized. I can revert the removal here, but maybe we can create a task to round up all existing uses of the flags and work with the original code authors to get them removed?